lib/credentials.ts

```diff
@@ -90,10 +90,6 @@ export const issueCompanyCredential = async (
   };
   try {
     const credential = await issueCredential(rawCredential);
-    await setAddressRoleInDb(
```

Comment:
This call is being made from front-end, hence moved this database call to the API call instead of here.